### PR TITLE
LPD-7839 Create a Test to Cover Navigate to Edit Object Definition Page

### PR DESCRIPTION
**Pull request by igor-franca**
Related [Ticket](https://liferay.atlassian.net/browse/LPD-25461)

### CODE REVIEW

File: modules/test/playwright/tests/object-web/objectDefinitions.spec.ts [hunk 1]
```diff
@@ -273,6 +273,32 @@ test.describe('Manage object definitions through Model Builder', () => {
...
+
+		await modelBuilderPage.openPageViewButton.click();
+
+		const editObjectDefinitionPage = await pagePromise;
+
+		await expect(
+			editObjectDefinitionPage.getByText('ERC:L_ORGANIZATION')
+		).toBeVisible();
+	});
+
```
Comment (igor-franca) [inline modules/test/playwright/tests/object-web/objectDefinitions.spec.ts:302]: [reference](https://playwright.dev/docs/pages#handling-new-pages)

File: modules/test/playwright/tests/object-web/objectDefinitions.spec.ts [hunk 2]
```diff
@@ -273,6 +273,32 @@ test.describe('Manage object definitions through Model Builder', () => {
 		await apiHelpers.objectAdmin.deleteObjectFolder(objectFolder.id);
 	});
 
+	test('navigate to edit object definition page', async ({
+		context,
+		modelBuilderPage,
+		page,
+	}) => {
+		const pagePromise = context.waitForEvent('page');
```
Comment (cgmonte) [inline modules/test/playwright/tests/object-web/objectDefinitions.spec.ts:284]: You only need to start waiting for this event right before calling `openPageViewButton.click():`. Then we would have something like this:
```
const pagePromise = context.waitForEvent('page');

await modelBuilderPage.openPageViewButton.click();

const editObjectDefinitionPage = await pagePromise;
```

**Review by cgmonte — CHANGES_REQUESTED** [review]
Hi @igor-franca, I only have a small request for change. Please, take look and let me known what you think. Thanks

**Comment by igor-franca** [comment]
ci:test:sf

**Review by cgmonte — APPROVED** [review]
LGTM!

**Comment by igor-franca** [comment]
SF reported the following error:

```
     [exec] Caused by: java.net.ConnectException: Connection refused (Connection refused)
     [exec] 	at org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:368)
     [exec] 	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
     [exec] 	... 245 more
```

I'll run it again and wait to check for this connection error.

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
> There appears to be an issue with Jenkins nodes connecting to repository-cdn. I’ve contacted IT . Hopefully they can resolve this issue soon.

-Peter Yoo, [here](https://liferay.slack.com/archives/C19PWQ74J/p1715610465771059?thread_ts=1715598586.723149&cid=C19PWQ74J)

**Comment by igor-franca** [comment]
ci:test:sf

**Comment by igor-franca** [comment]
ci:test:playwright

**Comment by igor-franca** [comment]
ci:forward

**Pull request closed without merging**